gluon/contrib/redis_scheduler.py

```diff
@@ -31,7 +31,7 @@
 from gluon.contrib.redis_scheduler import RScheduler
 
 def demo1(*args,**vars):
-    print 'you passed args=%s and vars=%s' % (args, vars)
+    print('you passed args=%s and vars=%s') % (args, vars)
```

Comment:
I find this cleaner : 
print('you passed args=%s and vars=%s' % (args, vars))

Or even : 
print('you passed args={} and vars={}'.format(args, vars))